contracts/lib/time_series/tests/test_extract_fields.cairo

```diff
@@ -0,0 +1,39 @@
+%lang starknet
+
+from starkware.cairo.common.alloc import alloc
+
+from time_series.extract_fields import transform_arr
+from time_series.structs import TickElem
+
+struct DataPoint:
+    member timestamp : felt
+    member val1 : felt
+    member val2 : felt
+end
```

Comment:
How does this differ from `TickElem`

Reply:
What are `val1` and `val2`

Reply:
it has an additional field.  val1 and val2 could be anything your imagination wants them to be.  It's a struct used to test the extraction of field elements.  The only important thing is that it was not in the exact same format as a TickElem so a custom transformation could be tested.